spec/models/user_spec.rb

```diff
@@ -87,13 +89,53 @@
   end
 
   describe 'methods' do
+    describe '#attended_event?' do
+      context 'user has attended to the event' do
+        before { events_registration.update_attributes(attended: true) }
```

Comment:
I would prefer writing this with `do end` instead `{ }`, as it is how it appears in the rspec documentation